money-core/src/main/scala/com/comcast/money/core/Formatters.scala

```diff
@@ -21,6 +21,21 @@ import com.comcast.money.api.SpanId
 import scala.util.Try
 
 object Formatters {
+  implicit class StringHexHelpers(s: String) {
```

Comment:
This may want to be private and I would prefer a name like `StringWithB3HexConversion` or something less generic than helpers.

the below shows some naming conventions
https://docs.scala-lang.org/overviews/core/implicit-classes.html